src/VisualStudio/Core/Def/Implementation/ChangeSignature/ChangeSignatureDialog.xaml.cs

```diff
@@ -153,9 +171,14 @@ private void Add_Click(object sender, RoutedEventArgs e)
                     addParameterViewModel.TypeBinds);
 
                 _viewModel.AddParameter(addedParameter);
-            }
 
-            SetFocusToSelectedRow();
+                SetFocusToSelectedRow(true);
```

Comment:
Interesting. I would have thought this would passed `false` to leave the focus wherever it was. Let's discuss.